main.go

```diff
@@ -129,7 +132,22 @@ func main() {
 		}
 		return nil
 	}
+	// If the command returns an error, cli takes upon itself to print
+	// the error on cli.ErrWriter and exit.
+	// Use our own writer here to ensure the log gets sent to the right location.
+	originalCliErrWriter = cli.ErrWriter
+	cli.ErrWriter = &FatalWriter{}
```

Comment:
nit: instead of the global `originalCliErrWriter` add it to a field on `FatalWriter`